### PR TITLE
standalone: space in ffmpeg path dir name issue 

### PR DESCRIPTION
**Pull request by jezscha**

### CODE REVIEW

**Review by iLLiCiTiT — APPROVED** [review]
Approved

File: pype/plugins/standalonepublisher/publish/extract_trim_video_audio.py [hunk 1]
```diff
@@ -60,7 +60,7 @@ def process(self, instance):
                 ]
 
             args = [
-                ffmpeg_path,
+                f"\"{ffmpeg_path}\"",
```
Comment (iLLiCiTiT) [inline pype/plugins/standalonepublisher/publish/extract_trim_video_audio.py:63]: WAIT

Comment (iLLiCiTiT) [reply pype/plugins/standalonepublisher/publish/extract_trim_video_audio.py:63]: This is Python 3. That would not work in hiero!

Comment (jezscha) [reply pype/plugins/standalonepublisher/publish/extract_trim_video_audio.py:63]: this is standalone publisher

Comment (iLLiCiTiT) [reply pype/plugins/standalonepublisher/publish/extract_trim_video_audio.py:63]: Damn why do we have same plugin name in standalone publisher and in global...

**Review by iLLiCiTiT — APPROVED** [review]
Approved

**Pull request merged**